.flowconfig

```diff
@@ -1,3 +1,4 @@
 [ignore]
 .*/node_modules/postcss-import/.*
+.*/node_modules/findup/test/fixture.*
```

Comment:
This was added to prevent a flow error where the test JSON files in the `findup` package are empty and would through `Unexpected end of input` errors